nuget.config

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="daily" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
```

Comment:
Looks good. We should consider changing this to add a local folder as the nuget feed to restore the package from and check-in the package into the repo. That way, we get the custom intellisense xml (without having to round-trip via docs which takes a non-trivial amount of time).

Reply:
Agreed. Let's do this later today when we're crafting the package.